### ISSUE TITLE
persons not showing

### ISSUE THREAD

**Issue by martinantonmueller** [acdh-oeaw/schnitzler-tagebuch#39]
This happened to me a few times the last couple of days so it is not a single instance. Only this time I made a note. »Herr Hietler« (not the one from Monty Python) is mentioned in the diary on 1883-09-02

PMB:
https://pmb.acdh.oeaw.ac.at/apis/entities/entity/person/18259
has this URI for the diary:
https://schnitzler-tagebuch.acdh.oeaw.ac.at/entity/person_16190

but when one uses the uri it says: "Kein Eintrag gefunden". Nevertheless the now only relevant file index_person_day.xml has him listed properly:

<item target="1883-09-02">
      <ref>person_14895</ref>
      <ref>person_16371</ref>
      <ref>person_16372</ref>
      <ref>person_18600</ref>
      <ref>person_16190</ref>
      <ref>person_18322</ref>
      <ref>person_14898</ref>
   </item>

@csae8092 what's your opinion?

**Comment by csae8092** [comment]
yes the app was deployed without denormalizing the index, I'll look into this in a couple of days
